texts/Bade Bhaisaab (disambiguated) .txt

```diff
@@ -0,0 +1,1586 @@
+"<बड़े>"
+	"*बड़े"
+"<थे>"
+	"हो" vbser impf p3 m pl
+"<।>"
+	"।" sent
+"<मेरी>"
+;	"मेरी" np ant f sg nom
+	"मैं" prn pers p1 f sg gen
+"<उम्र>"
+;	"उम्र" n f sg nom
+	"उम्र" n f sg obl
+;	"उम्र" n f pl nom
+;	"उम्र" n f pl obl
+"<नौ>"
+	"नौ" num
+"<साल>"
+	"साल" n m sg nom
+;	"साल" n m sg obl
+;	"साल" n m pl nom
+"<की>"
+	"का" gen f sg
+;	"का" gen f pl
+;	"कर" vblex tv perf f sg
+;	"कर" vblex tv perf f pl
+"<थी>"
+	"हो" vbser impf p3 f sg
+"<,>"
+	"," cm
+"<वह>"
+	"यह" prn dem p3 mf sg dist nom
+"<चौदह>"
+	"चौदह" num
+;	"चौदह" adj mfn sp
+;	"चौदह" adj m sg nom
+;	"चौदह" adj m sg obl
+;	"चौदह" adj m pl nom
+;	"चौदह" adj m pl obl
+;	"चौदह" adj f sg nom
+;	"चौदह" adj f sg obl
+;	"चौदह" adj f pl nom
+;	"चौदह" adj f pl obl
+"<साल>"
+	"साल" n m sg nom
+;	"साल" n m sg obl
+;	"साल" n m pl nom
+
+"<के>"
+	"का" gen m pl
+"<थे>"
+	"हो" vbser impf p3 m pl
+"<।>"
+	"।" sent
+"<उन्हें>"
+	"यह" prn dem p3 mf pl dist dat
+"<मेरी>"
+;	"मेरी" np ant f sg nom
+	"मैं" prn pers p1 f sg gen
+"<तंबीह>"
+	"*तंबीह"
+"<और>"
+	"और" cnjcoo
+"<निगरानी>"
+;	"निगरानी" n f sg nom
+	"निगरानी" n f sg obl
+;	"निगरानी" n f pl nom
+;	"निगरानी" n f pl obl
+"<का>"
+	"का" gen m sg
+"<पूरा>"
+;	"पूरा" adj mfn sp
+	"पूरा" adj m sg nom
+;	"पूरा" adj mfn sp obl
+"<जन्मसिद्ध>"
+;	"जन्मसिद्ध" adj mfn sp
+	"जन्मसिद्ध" adj m sg nom
+;	"जन्मसिद्ध" adj m sg obl
+;	"जन्मसिद्ध" adj m pl nom
+;	"जन्मसिद्ध" adj m pl obl
+;	"जन्मसिद्ध" adj f sg nom
+;	"जन्मसिद्ध" adj f sg obl
+;	"जन्मसिद्ध" adj f pl nom
+;	"जन्मसिद्ध" adj f pl obl
+"<अधिकार>"
+	"अधिकार" n m sg nom
+;	"अधिकार" n m sg obl
+;	"अधिकार" n m pl nom
+"<था>"
+	"हो" vbser impf p3 m sg
+"<।>"
+	"।" sent
+"<और>"
+	"और" cnjcoo
+"<मेरी>"
+;	"मेरी" np ant f sg nom
+	"मैं" prn pers p1 f sg gen
+"<शालीनता>"
+;	"शालीनता" n f sg nom
+	"शालीनता" n f sg obl
+;	"शालीनता" n f pl nom
+;	"शालीनता" n f pl obl
+"<इसी>"
+;	"इसी" adj mfn sp
+;	"इसी" adj m sg nom
+;	"इसी" adj m sg obl
+;	"इसी" adj m pl nom
+;	"इसी" adj m pl obl
+;	"इसी" adj f sg nom
+	"इसी" adj f sg obl
+;	"इसी" adj f pl nom
+;	"इसी" adj f pl obl
+"<में>"
+	"में" post
+"<थी>"
+	"हो" vbser impf p3 f sg
+"<कि>"
+	"कि" cnjsub
+"<उनके>"
+	"यह" prn dem p3 mf pl dist gen
+"<हुक्म>"
+;	"हुक्म" n m sg nom
+	"हुक्म" n m sg obl
+;	"हुक्म" n m pl nom
+"<को>"
+	"को" post
+"<कानून>"
+	"कानून" n m sg nom
+;	"कानून" n m sg obl
+;	"कानून" n m pl nom
+"<समझूँ>"
+	"समझ" vblex tv prs p1 sg
+;	"समझ" vblex iv prs p1 sg
+"<।>"
+	"।" sent
+"<.>"
+	"." sent
+
+
+"<वह>"
+	"यह" prn dem p3 mf sg dist nom
+"<स्वभाव>"
+	"स्वभाव" n m sg nom
+;	"स्वभाव" n m sg obl
+;	"स्वभाव" n m pl nom
+"<से>"
+	"से" post
+;	"सा" adj m sg obl
+;	"सा" adj m pl nom
+;	"सा" adj m pl obl
+"<बड़े>"
+	"*बड़े"
+"<अघ्ययनशील>"
+	"*अघ्ययनशील"
+"<थे>"
+	"हो" vbser impf p3 m pl
+"<।>"
+	"।" sent
+"<हरदम>"
+	"*हरदम"
+"<किताब>"
+	"किताब" n f sg nom
+;	"किताब" n f sg obl
+"<खोले>"
+;	"खोल" vblex tv prs p2 sg
+	"खोल" vblex tv prs p3 sg
+;	"खोल" vblex tv perf m pl
+"<बैठे>"
+;	"बैठ" vblex iv prs p2 sg
+	"बैठ" vblex iv prs p3 sg
```

Comment:
This is not really the `prs` (present tense) conjugation, it is the perfective conjugation since it is followed by रहते. You can look up the Wikipedia articles on perfective aspect, tense, etc. if this is a little confusing.